docs/src/prototypes/CopyToClipboard/index.tsx

```diff
@@ -27,27 +27,37 @@ const CopyToClipboardPrototype: React.FC<CopyToClipboardPrototypeProps> = props
 }
 
 const CopyToClipboardInMenu: React.FC = props => {
+  const targetRef = React.useRef<HTMLElement>(null)
+
+  const item = {
+    key: 'edit',
+    content: 'Edit',
+    menu: [
+      'Open File...',
+      'Save File...',
+      render =>
+        render('Copy text', (Component, props) => {
+          return (
+            <CopyToClipboard
+              targetRef={targetRef}
+              position="after"
+              align="bottom"
+              pointing
+              value={'Julius Caesar'}
+              trigger={<Component {...props} />}
+            />
+          )
+        }),
+    ],
+  }
+
   const items = [
-    {
-      key: 'edit',
-      content: 'Edit',
-      menu: [
-        'Open File...',
-        'Save File...',
-        render =>
-          render('Copy text', (Component, props) => {
-            return (
-              <CopyToClipboard
-                position="after"
-                align="bottom"
-                pointing
-                value={'Julius Caesar'}
-                trigger={<Component {...props} />}
-              />
-            )
-          }),
-      ],
-    },
+    render =>
+      render(item, (Component, props) => (
+        <Ref innerRef={targetRef}>
```

Comment:
https://github.com/stardust-ui/react/pull/1900/files#r322141672

```suggestion
        <Ref innerRef={setTarget}>
```